tests/lib_test.rs

```diff
@@ -590,3 +590,241 @@ fn account_credential_deployment_signed_payload() {
     );
 }
+
+#[test]
+fn identity_issuance_request_parameters_debug() {
+    assert_eq!(
```

Comment:
It gets very hard (for me at least) to read these very long inline equality checks. Would be nice if we could do something like

```rust
let actual = ...;
let expected = ...;
assert_eq(actual, expected); // or assert_eq(format("{:?}", actual), expected);
```